rr_common/src/follower/follower.cpp

```diff
@@ -92,6 +106,8 @@ int main(int argc, char** argv) {
     nhp.param("setpoint_topic", setpoint_topic, string("/setpoint"));
     nhp.param("topic_from_controller", topic_from_controller, string("/control_effort"));
 
+    visonBox_pub = nh.advertise<geometry_msgs::PolygonStamped>("vision_box", 1);
```

Comment:
Also, add `follower/` here either by just adding `follower/vision_box` topic name with nh or just `vision_box` with nhp which is the node's private node handle.